### PR TITLE
fix:When the label type is equal to the line, when marking the starting point, directly clicking on the outside of the picture will produce an error

### PR DESCRIPTION
**Pull request by xiecz123**

### CODE REVIEW

File: src/logic/render/LineRenderEngine.ts [hunk 1]
```diff
@@ -58,6 +58,8 @@ export class LineRenderEngine extends BaseRenderEngine {
                 store.dispatch(updateActiveLabelId(labelLineUnderMouse.id));
             } else if (!this.isInProgress() && isMouseOverImage) {
                 this.startNewLabelCreation(data)
+            } else if (!this.isInProgress()) {
+
             } else {
```
Comment (SkalskiP) [inline src/logic/render/LineRenderEngine.ts:63]: Thank you for pointing out this bug. I would suggest another solution to this problem. We will correct this line.
```suggestion
            } else if (isMouseOverImage) {
```
I guess it should work?

Comment (xiecz123) [reply src/logic/render/LineRenderEngine.ts:63]: This will lose a feature（The ending point can be outside the picture and in canvas），you can try it

Comment (xiecz123) [reply src/logic/render/LineRenderEngine.ts:61]: another way
```diff
-            } else if (!this.isInProgress() && isMouseOverImage) {
-                this.startNewLabelCreation(data)
+            } else if (!this.isInProgress()) {
+                isMouseOverImage && this.startNewLabelCreation(data)
```

Comment (SkalskiP) [reply src/logic/render/LineRenderEngine.ts:61]: I'll take a look at it ever the weekend and provide some suggestion :) (or just agree with you)

Comment (SkalskiP) [reply src/logic/render/LineRenderEngine.ts:61]: Ok I took a look, and I would propose changing line 61 into: 
```suggestion
            } else if (this.isInProgress()) {
```

Comment (Bernstein7z) [reply src/logic/render/LineRenderEngine.ts:61]: #167

**Comment by SkalskiP** [comment]
I proposed alternative solution in: https://github.com/SkalskiP/make-sense/pull/194

**Pull request closed without merging**